src/org/wof/foodministry/FoodEntry.java

```diff
@@ -28,5 +28,29 @@ public String getDay() {
     
     public void setDay ( String day ) {
         this.day = day;
-    } 
+    }
+    public int getweight() {
+	return weight;
+    }
+    public void setweight ( int weight ) {
+        this.weight = weight;
+    }
+    public int getDate () {
+	return date;
+    }
+    public void setDate ( int date ) {
+        this.date = date;
+    }
+    public String getTime() {
+	return Time;
+    }
+    public void setTime() {
+        this.Time =Time;
+    }
+    public String getLocation() {
```

Comment:
It looks like getLocation and setLocation are new methods. This makes sense because you get the food from different stores. I like it.

My question though is if you create a new FoodEntry object, where would you define the Location in the class constructor for the new object?
```
FoodEntry(String day, String time, int weight, int waste) {
 	this.day = day;
 	this.time = time;
 	this.weight = weight;
 	this.waste = waste;
}```